### DISCUSSION TITLE
I don't know how to implement the functionality of toggling and displaying tags like in the tutorial documentation.

### DISCUSSION THREAD

**Discussion by dogsking** [requarks/wiki#6434] [Help / Questions] [answered]
![1](https://github.com/requarks/wiki/assets/52346183/b62fdf2b-9dc8-4944-a78e-71926f6e408a)
I would like to create a toggle switch functionality with the same style as shown in the image, but I seem to have not found it in the documentation. I am using a markdown page.

**Comment by mittkkoo** [comment, marked as answer]
```markdown
## Some caption {.tabset}

### Tab 1
sjfsjufol

### Tab 2
josujfdsujf
```
